src/jwt.cc

```diff
@@ -23,6 +24,19 @@
 namespace google {
 namespace jwt_verify {
 
+bool isImplemented(absl::string_view alg) {
```

Comment:
please add annoymous namespace for this function.

Reply:
Added in 0eac53f!